x-pack/plugin/ml/src/test/java/org/elasticsearch/xpack/ml/MachineLearningTests.java

```diff
@@ -97,8 +97,8 @@ public void testNoAttributes_givenClash() {
 
     public void testMachineMemory_givenStatsFailure() throws IOException {
         OsStats stats = mock(OsStats.class);
-        when(stats.getMem()).thenReturn(new OsStats.Mem(-1, -1));
-        assertEquals(-1L, MachineLearning.machineMemoryFromStats(stats));
+        when(stats.getMem()).thenReturn(new OsStats.Mem(0, 0));
+        assertEquals(0L, MachineLearning.machineMemoryFromStats(stats));
```

Comment:
I think a corresponding change of `mem < 0` to `mem <= 0` should also be made in the `MachineLearning.machineMemoryFromStats()` method, on lines 804 and 805 of `MachineLearning.java`.

Reply:
I've updated that also, thanks for the heads up!